server/index.js

```diff
@@ -0,0 +1,23 @@
+const thoughtsRoutes = require('./routes/thought')
+const express = require('express');
+const cors = require('cors');
+const server = express();
+const bodyParser = require("body-parser")
+server.use(cors());
+server.use(express.json());
+server.use(bodyParser.json())
+const { Pool } = require("pg");
+const pool = new Pool()
+
+const port = process.env.PORT || 3050;
```

Comment:
in your docker-compose.yaml file, you have stated that the node container is running locally on port 3000. The server does not start up because the port you have specified here does not match up with that. Changing this to 3000 fixes the issue.